cmd/kyma/run/function/function.go

```diff
@@ -119,7 +121,7 @@ func (c *command) runContainer(ctx context.Context, client *client.Client, cfg w
 	}
 
 	step.Successf(fmt.Sprintf("Runned container: %s", c.opts.ContainerName))
```

Comment:
Sorry, I just saw this line... I know you did not change it, but we can not make such spelling errors. It gives a very poor image to the users. 
@kazydek just a heads up that in the CLI there are lots of printed messages in the code that also would need TW review, not only the docs!

Additionally, the `step.Successf()` (no need for `fmt.Sprintf`) function is being used wrongly. Correction for both:
```suggestion
	step.Successf("Ran container: %s", c.opts.ContainerName)
```

Finally, line 120 has a misspelling too:
`return errors.Wrap(err, "white trying to run container")`